Add Windows 10 Screenshots (#797)
wpilibsuite/frc-docs: source/docs/networking/networking-introduction/windows-firewall-configuration.rst

.. tabs::

   .. group-tab:: Windows 10

      Click :guilabel:`Start` -> :guilabel:`Settings`

      .. image:: images/windows-firewall/select-settings.png

      Click :guilabel:`Update & Security`

      .. image:: images/windows-firewall/select-update-security.png

      In the left pane, select :guilabel:`Open Windows Security`

      .. image:: images/windows-firewall/open-windows-security.png

      In the left pane, select :guilabel:`Firewall and network protection`

      .. image:: images/windows-firewall/select-firewall.png

      Click on **each** of the highlighted options

      .. image:: images/windows-firewall/select-network.png

      Then click on the **On** toggle to turn it off.

      .. image:: images/windows-firewall/disable-firewall-toggle.png

   .. group-tab:: Windows 7

      .. image:: images/windows-firewall-configuration-1.png

      Click :guilabel:`Start` -> :guilabel:`Control Panel` to open the Control Panel. Click the dropdown next to :guilabel:`View by:` and select :guilabel:`Small icons` then click :guilabel:`Windows Defender Firewall`.

      .. image:: images/windows-firewall-configuration-2.png

>       In the left pane, click :guilabel`Turn Windows Defender Firewall on or off`, and click yes. Enter your Administrator password if a dialog appears.

Reviewer: ```suggestion
      In the left pane, click :guilabel:`Turn Windows Defender Firewall on or off`, and click yes. Enter your Administrator password if a dialog appears.
```

Disabling Windows Firewall
--------------------------

> .. important:: Disabling your firewall requires administrator privelages to the PC. Additionally note that disabling the firewall is not recommended for computers that often connect to the internet.

Reviewer: ```suggestion
.. important:: Disabling your firewall requires administrator privileges to the PC. Additionally note that disabling the firewall is not recommended for computers that connect to the internet.
```

   .. group-tab:: Windows 10

      Click :guilabel:`Start` -> :guilabel:`Settings`

      .. image:: images/windows-firewall/select-settings.png

      Click :guilabel:`Update & Security`

      .. image:: images/windows-firewall/select-update-security.png

      In the left pane, select :guilabel:`Open Windows Security`

      .. image:: images/windows-firewall/open-windows-security.png

      In the left pane, select :guilabel:`Firewall and network protection`

      .. image:: images/windows-firewall/select-firewall.png

      At the bottom of the window, select :guilabel:`Allow an app through firewall`

      .. image:: images/windows-firewall/add-to-firewall.png

      For each FRC program you are having an issue with, make sure that it appears in the list and that it has a check in each of the 3 columns. If you need to change a setting, you made need to click the :guilabel:`Change settings` button in the top right before changing the settings. If the program is not in the list at all, click the :guilabel:`Allow another program...` button and browse to the location of the program to add it.

>       .. image:: images/windows-firewall-configuration-6.png

Reviewer: Windows 10 screenshot?

Author: It's the same look on Windows 10

Reviewer: Mine looks slightly different. However, even if the content is the same, the the theme change is jarring

   .. group-tab:: Windows 7

>       Click :guilabel:`Start` -> :guilabel`Control Panel` to open the Control Panel. Click the dropdown next to :guilabel:`View by:` and select :guilabel:`Small icons` then click :guilabel:`Windows Defender Firewall`.

Reviewer: ```suggestion
      Click :guilabel:`Start` -> :guilabel:`Control Panel` to open the Control Panel. Click the dropdown next to :guilabel:`View by:` and select :guilabel:`Small icons` then click :guilabel:`Windows Defender Firewall`.
```
